src/main/java/net/sf/jabref/gui/cleanup/FieldFormatterCleanupsPanel.java

```diff
@@ -193,8 +217,7 @@ private JPanel getSelectorPanel() {
                         "pref, 2dlu, pref:grow, 2dlu"));
 
         List<String> fieldNames = InternalBibtexFields.getAllPublicFieldNames();
-        fieldNames.add(BibEntry.KEY_FIELD);
-        fieldNames.add("all");
+        fieldNames.addAll(Arrays.asList(BibEntry.KEY_FIELD, FieldName.ABSTRACT_ALL_FIELD, FieldName.ABSTRACT_ALL_TEXT_FIELDS_FIELD));
```

Comment:
I would do three calls using `fieldNames.add()`. This is IMHO more readable than `Arrays.asList`